applications/subghz/views/subghz_frequency_analyzer.c

```diff
@@ -20,11 +37,20 @@ struct SubGhzFrequencyAnalyzer {
     SubGhzFrequencyAnalyzerCallback callback;
     void* context;
     bool locked;
+    float rssi_last;
+    uint32_t frequency_last;
```

Comment:
I think this does not solve the underlying issue.

If one is scanning a busy area, the "last" would flicker just as much as the "current", making it unreadable the same way as the existing implementation.

#opinionated

Reply:
https://github.com/flipperdevices/flipperzero-firmware/issues/1466

Reply:
> I think this does not solve the underlying issue.
> 
> If one is scanning a busy area, the "last" would flicker just as much as the "current", making it unreadable the same way as the existing implementation.
> 
> #opinionated

No, it's updated with the last value after there is no signal for a while.

Reply:
Ah. I see. Well, the last time I used c was about 15 years ago 😅 
Anyhow, it would be great to reuse the `subghz_history` and utilize the up and down arrows to navigate in the history. What do you think?
I have been trying to do just that, but unfortunately c was really really a long time ago.. 😞

Reply:
> Anyhow, it would be great to reuse the `subghz_history` and utilize the up and down arrows to navigate in the history. What do you think?

I think that it's good idea but i'm not ready to implement it now. Maybe later.

Reply:
Hello, 

What's the best way to contribute to the "history feature" ? 
Waiting for this PR to be validated ? Forking your fork & PR to it ?

Reply:
> Hello,
> 
> What's the best way to contribute to the "history feature" ? Waiting for this PR to be validated ? Forking your fork & PR to it ?

Not sure. Let's wait for PR to be validated.

Reply:
Damn I wish I was better at cpp! 😅